src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/PackageRestore/Snapshots/RestoreBuilderTests.cs

```diff
@@ -62,6 +62,27 @@ public void ToProjectRestoreInfo_WhenNuGetRestoreRuleMissing_ReturnsEmpty()
             Assert.Empty(targetFramework.Properties);
         }
 
+        [Fact]
+        public void ToProjectRestoreInfo_RespectsNuGetTargetMonikerIfPresent()
```

Comment:
Minor: Cross ref GitHub issue here for future reference: https://github.com/dotnet/project-system/issues/4854

Reply:
You don't think history gives you that?

Reply:
Sure it does. Just not as convenient.